includes/class-wsuwp-plugin-idonate-restapi.php

```diff
@@ -119,4 +120,35 @@ function wsuf_fundselector_funds_get_fund_by_des_id_rest( $data ) {
 
 		return $this->fundselector_shortcode->wsuf_fundselector_funds_get_fund_name( $designation_id );
 	}
+
+	/**
+	* Add a new custom REST endpoint to get a list of funds based off of a search term
+	*
+	* @since 1.1.2
+	**/
+	function wsuf_fundselector_register_endpoint_fund_search() {
```

Comment:
@philcable Is filter[s] an actual query parameter we can use? I tried to find some documentation on how to use it, but couldn't get any useful results. I had tried using the default endpoint with ?status=searchable, but for the default endpoint I was getting errors that made me think that the request had to be authenticated first. The custom endpoint let us workaround the authentication issue.